src/components/ButtonGroup/ButtonGroup.jsx

```diff
@@ -107,7 +108,8 @@ const ButtonGroup = React.createClass(createLucidComponentDefinition({
 							isActive={_.includes(selectedIndices, index)}
 							{...buttonChildProp}
 							key={index}
-							onClick={_.partial(this.handleSelect, index, buttonChildProp.onClick)}
+							uniqueId={index}
```

Comment:
It might be confusing if you pass in `<ButtonGroup.Button uniqueId='foo'>` and when they click it, it comes back with an index. Maybe we should default to index, but use a passed in `uniqueId` if they specify.